### ISSUE TITLE
There is no release

### ISSUE THREAD

**Issue by Sayuris1** [defold/sample-lights-and-shadows#3]
There is no specific release currently so we need to add the main branch as a dependency.

**Comment by JCash** [comment]
It's a sample, and not really intended as a resource to be dependent on.

**Comment by britzl** [comment]
> It's a sample, and not really intended as a resource to be dependent on.

This is true. But I also created in such a way that it is possible to use it as a dependency. So the suggestion by @Sayuris1 is valid I think.

**Comment by JCash** [comment]
Then I think the naming is a misleading and we haven't discussed the support burden or the design of "sample" projects.

There are currently 9 "sample" projects, and afaik, we have never treated them as supported dependencies.
https://github.com/orgs/defold/repositories?q=sample&type=all&language=&sort=

As for using it as a dependency, it's easy enough to link to any version, clicking the commit you want:
E.g. [the latest dependency](https://github.com/defold/sample-lights-and-shadows/archive/ad63ffcbf489859b363302d9e2b82083b9c4d829.zip)

**Comment by Sayuris1** [comment]
I didn't know we can link the commit as a dependency. I think that solves the problem. Lights are pretty usable as a module currently.

**Comment by paulbaucom** [comment]
I agree this is very usable as a module and would be beneficial in the asset portal and upgraded from "sample" status.  That is assuming there isn't a better solution for this type of 2D lighting effect already there (let me know if there is).  I think the ball sample should be included as well.  It would look good on the asset portal page for new devs.

I am working on a fork of this since I found a typo/bug in the lights.lua (#11) and fixed it for a project I used this module in.  I also found that there is an issue with rendering on AMD cards.  Ref forum post: https://forum.defold.com/t/2d-lights-and-shadows-sample/68970/21?u=pabaucom

I am trying to fix this or come up with a workaround (I will also test with my intel iGPU).  I would also like to cleanup/update the code.  I merged changes from the current default.render_script into the lights.render_script.

Do you think this is time well spent for this at this point?  Should I continue to submit PRs/issues for this repo?

**Comment by paulbaucom** [comment]
It appears I have fixed the shadow issue on my AMD card.  I opened issue #12 where I propose a fix.

**Comment by britzl** [comment]
> I opened issue #12 where I propose a fix.

Please create a PR with this fix so that we can review and discuss it!
